### DISCUSSION TITLE
Any programmatic API documentation?

### DISCUSSION THREAD

**Discussion by GoldJohnKing** [sandboxie-plus/Sandboxie#1203] [Documents and notes]
Hi, is there any API that I can use to control SbieDriver?

For example, I would like to write a program to create a sandbox, launch process inside the sandbox, and then destroy the sandbox.

Is there any way that I can implement it by intereacting with SbieDriver or Sandboxie-Plus?

**Comment by GoldJohnKing** [comment]
I see these in original Sandboxie docs:
https://sandboxie-website-archive.github.io/www.sandboxie.com/StartCommandLine.html
https://sandboxie-website-archive.github.io/www.sandboxie.com/SBIE_DLL_API.html

This sounds good to me, however, are those methods still available in Sandboxie-Plus? And is there any new API or command line not listed there?

**Comment by isaak654** [comment]
1) You may want to read [this](https://github.com/sandboxie-plus/Sandboxie#features) paragraph that offers an explanation about the distinctions between Sandboxie Classic and Plus.
2) If you need more command lines about the sandboxes, I woud suggest to check [this](https://github.com/sandboxie-plus/Sandboxie/issues/278#issuecomment-856207910) and [this](https://sandboxie-website-archive.github.io/www.sandboxie.com/old-forums/viewtopica6bca6bc.html#p126947). SbieIni.exe is included in both Classic and Plus.

If you want to contribute to the [Sandboxie documentation](https://sandboxie-plus.github.io/sandboxie-docs) provided on GitHub, you can visit any page like [this](https://sandboxie-plus.github.io/sandboxie-docs/Content/SandboxieIni.html) and click on the "Improve this page" link at the bottom of the page to start a pull request.
